File: 2023-2024/sourcecode/mod4/lab12/RetrofitBasicsJava/app/src/main/java/com/example/retrofitbasicsjava/data/api/PlaceholderApi.java
package com.example.retrofitbasicsjava.data.api;

import com.example.retrofitbasicsjava.data.Post;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface PlaceholderApi {
    @GET("posts")
    Call<List<Post>> posts();
}

File: 2023-2024/sourcecode/mod4/lab12/ResourcePatterinBasicsCompose/app/src/main/java/com/example/resourcepatterinbasicscompose/api/PlaceholderApi.kt
package com.example.resourcepatterinbasicscompose.api

import com.example.resourcepatterinbasicscompose.data.CommentResponseItem
import retrofit2.Response
import retrofit2.http.GET

interface PlaceholderApi {
    @GET("comments")
    suspend fun comments(): Response<List<CommentResponseItem>>
}